starknet_py/net/account/account.py

```diff
@@ -344,6 +404,56 @@ async def sign_invoke_v1(
         signature = self.signer.sign_transaction(execute_tx)
         return _add_signature_to_transaction(execute_tx, signature)
 
+    async def sign_outside_execution_call(
```

Comment:
```suggestion
    async def sign_outside_transaction(
```

Reply:
@franciszekjob 
I would oppose that, this is misleading. Real transaction will be signed and sent during the execution. This is not a transaction object, it leaves outside transactions.py package. JS library adopted that syntax for naming, but I disagree. What do you think? I'll only change if you insist.

Reply:
@baitcode 

> Real transaction will be signed and sent during the execution.

I'm not sure that's correct tbh. Inside `sign_outside_execution_call` we do use `sign_message`. What do you think? I'm open for discussion :D

Reply:
>> I'm not sure that's correct tbh. 
What confuses you?
>> we do use sign_message
which also points that this method does not create (or prepare) a transaction, as we use `sign_transaction` there.

I find `sign_outside_execution_call` perfect. You clearly understand that you produce call, you understand that it is for outside execution, and you then use it as a call in other account methods.

Reply:
Ok, fine, let's leave it like that.

Reply:
we can rename to `prepare_outside_execution_call` hiding the fact that we sign anything. Which might be a tiny bit better or `create_outside_execution_call` which will create a clear distinction from transaction API.